File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -988,7 +837,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Form authentication
 
-The Blink HTML converter provides support for form authentication by using cookies. The cookies are send to web server for form authentication when the HTML page is requested. Each cookie is represented by a name and value. Refer to the following code snippet.
+The Blink HTML converter provides support for form authentication by [CookieCollection](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.CookieCollection.html) class using [Cookies](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_Cookies) property. The cookies are send to web server for form authentication when the HTML page is requested. Each cookie is represented by a name and value. Refer to the following code snippet.
```

Comment:
The Blink HTML converter provides support for form authentication by using [Cookies]() in [BlinkConverterSettings]() class

Reply:
Modified